DRAFT.md

```diff
@@ -1208,16 +1206,15 @@ constexpr polymorphic(allocator_arg_t, const Allocator& a,
 ```
 
 21. _Effects_: `allocator` is direct-non-list-initialized with `alloc`. If
-    `other` is valueless, `*this` is valueless. Otherwise, constructs an object
-    of type `polymorphic` that either owns the owned value of other, making
-    `other` valueless; or, owns an object of the same type constructed from
-    the owned value of `other` using the specified allocator, considering that
-    owned value as an rvalue.
-
-22. _[Note 1: This constructor does not require that `is_move_constructible_v<T>`
-  is `true`. --end note]_
-
-23. _[Note 2: The use of this function may require that `T` be a complete type
+    `other` is valueless, `*this` is valueless. Otherwise, if `alloc == other.alloc`
+    either constructs an object of type `polymorphic` that owns the owned value
+    of other, making`other` is valueless; or, owns an object of the same type
```

Comment:
Firstly looks like a space is missing, but also this does not ready quite right.  Should "making `other` is valueless" be "making `other` valueless" or "`other` is valueless" 
```suggestion
    of other, making `other` is valueless; or, owns an object of the same type
```

Reply:
Having re-read multiple times, I think this should be:
```suggestion
    of other, making `other` valueless; or, owns an object of the same type
```